config/crds/scheduling_v1alpha1_podgroup.yaml

```diff
@@ -41,3 +41,11 @@ spec:
           type: object
       type: object
   version: v1alpha1
+  subresources:
+    status: {}
+status:
```

Comment:
We could omit it if all fields are empty?

Reply:
if omit, `UpdateStatus` will not work :)

Reply:
Then it is OK. :) 

But in my CRD use case,
```
  subresources:
    status: {}
```
is enough.

Reply:
great to know, I'll remove the other part :)

Reply:
done